rust/operator-binary/src/hbase_controller.rs

```diff
@@ -736,33 +803,35 @@ fn build_rolegroup_statefulset(
             "pipefail".to_string(),
             "-c".to_string(),
         ])
-        .args(vec![format!(
-            "\
-mkdir -p {CONFIG_DIR_NAME}
-cp {HDFS_DISCOVERY_TMP_DIR}/hdfs-site.xml {CONFIG_DIR_NAME}
-cp {HDFS_DISCOVERY_TMP_DIR}/core-site.xml {CONFIG_DIR_NAME}
-cp {HBASE_CONFIG_TMP_DIR}/* {CONFIG_DIR_NAME}
-cp {HBASE_LOG_CONFIG_TMP_DIR}/{LOG4J_CONFIG_FILE} {CONFIG_DIR_NAME}
-
-{COMMON_BASH_TRAP_FUNCTIONS}
-{remove_vector_shutdown_file_command}
-prepare_signal_handlers
-bin/hbase {hbase_role_name_in_command} start &
-wait_for_termination $!
-{create_vector_shutdown_file_command}
-",
+        .args(vec![formatdoc! {"
+            mkdir -p {CONFIG_DIR_NAME}
+            cp {HDFS_DISCOVERY_TMP_DIR}/hdfs-site.xml {CONFIG_DIR_NAME}
+            cp {HDFS_DISCOVERY_TMP_DIR}/core-site.xml {CONFIG_DIR_NAME}
+            cp {HBASE_CONFIG_TMP_DIR}/* {CONFIG_DIR_NAME}
+            cp {HBASE_LOG_CONFIG_TMP_DIR}/{LOG4J_CONFIG_FILE} {CONFIG_DIR_NAME}
+
+            {kerberos_container_start_commands}
+
+            {COMMON_BASH_TRAP_FUNCTIONS}
+            {remove_vector_shutdown_file_command}
+            prepare_signal_handlers
+            bin/hbase {hbase_role_name_in_command} start &
+            wait_for_termination $!
+            {create_vector_shutdown_file_command}
+            ",
             hbase_role_name_in_command = match hbase_role {
                 HbaseRole::Master => "master",
                 HbaseRole::RegionServer => "regionserver",
                 // Of course it is not called "restserver", so we need to have this match
                 // instead of just letting the Display impl do it's thing ;P
                 HbaseRole::RestServer => "rest",
             },
```

Comment:
Not this PR, what about this?
```suggestion
            hbase_role_name_in_command = match hbase_role {
                HbaseRole::Master | HbaseRole::RegionServer => hbase_role.to_string(),
                // Of course it is not called "restserver", so we need to have this match
                // instead of just letting the Display impl do it's thing ;P
                HbaseRole::RestServer => "rest",
            },
```

Reply:
to_cli() sounds like a good thing, implemented in d3e1719